app/views/viewers/_universal_viewer.html.erb

```diff
@@ -1,4 +1,5 @@
-<div 
+<div
```

Comment:
This grabbed a fresh copy of master, which has the UV fix, but is being merged into #53, which doesn't have it. Ignore the stuff in this file

Reply:
Not sure why it's still showing up when I changed the target to master...